tests/integration/testutils/test_utils.go

```diff
@@ -35,6 +33,11 @@ const (
 	TruststoreFile     = "/certs/cassandra.truststore"
 )
 
+var (
+	// Hostname for the Cassandra service. (Will be the cassandra service inside the docker-compose file).
+	Hostnames = []string{"cassandra-3-11-0", "cassandra-4-0-3", "cassandra-latest-supported"}
+)
+
```

Comment:
I'd try to define this `Hostnames` and the ones defined in `cassandra_test.go` (`envCassandraVersions` and `cassandraContainerNames`) together.

Reply:
I have moved `Hostnames`, `envCassandraVersions`, `cassandraContainerNames ` together in the test_utils file as below

`var (
	CassandraConfigs = []struct {
		Version       string
		ContainerName string
		Hostname      string // Hostname for the Cassandra service. (Will be the cassandra service inside the docker-compose file).
	}{
		{
			Version:       "3.11.0",
			ContainerName: "cassandra-3-11-0",
			Hostname:      "cassandra-3-11-0",
		},
		{
			Version:       "4.0.3",
			ContainerName: "cassandra-4-0-3",
			Hostname:      "cassandra-4-0-3",
		},
		{
			Version:       "5.0.2",
			ContainerName: "cassandra-latest-supported",
			Hostname:      "cassandra-latest-supported",
		},
	}
)`

Also I have removed loop for every test implementation and added test cases to handle multiple version tests